src/lib/containers/table/SynapseTable.tsx

```diff
@@ -1202,6 +1218,7 @@ export default class SynapseTable extends React.Component<
   public applyChangesFromQueryFilter = (facets: FacetColumnRequest[]) => {
     const queryRequest: QueryBundleRequest = this.props.getLastQueryRequest!()
     queryRequest.query.selectedFacets = facets
+    this.setState({ isUserModifiedQuery: true })
```

Comment:
I understand what the variable name isUserModifiedQuery is getting at but I find it a bit confusing because the user could  modify the query in a number of ways, like with the old version of faceting or having sorted the columns of the data, but this variable doesn't apply to that.

 What do you think about renaming this variable to 'isQueryModifiedFromQueryFilter' or 'hasUserModifiedQueryWithQueryFilter'? I'm open to more concise suggestion as well

Reply:
@jay-hodgson has mentioned that we are going to have another way to modify the query along with the new filters.
May be it would make sense to base the naming on 'if the user can change the criteria'. Basically what we are getting at is if we want to allow them to change the selection criteria or not.  So may be 'canUserChangeCriteria' does that make more sense?

Reply:
I see more of what you mean now, it's a tricky name.. on second thought I think it's enough to just put a comment next to where the variable is declared and keep isUserModifiedQuery.